.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: install dependencies
+        run: npm ci
+      - name: run linter
+        run: npm run lint
+      - name: run build
+        run: npm run build
+
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: install dependencies
+        run: npm ci
+      - name: run tests
+        run: npm run test:e2e
```

Comment:
I am not sure if having a CI CD workflow without any test will make much sense at the moment.

Afterall, `test:e2e` script doesn't exist. So let's first get the initial features done and then add some test cases, then we can worry about CI CD workflows